diff --git a/notebooks_jason/max_of_K_all_models.py b/notebooks_jason/max_of_K_all_models.py
index 8288d67..435ff98 100644
--- a/notebooks_jason/max_of_K_all_models.py
+++ b/notebooks_jason/max_of_K_all_models.py
@@ -3712,6 +3712,10 @@ for var in (
     "cur_key_pairs",
     "category_name_remap",
     "subcubic_columns",
+    "ablation_data",
+    "ablation_data_by_key",
+    "d",
+    "all_tokens_datasets_lens",
 ):
     if var in locals():
         del locals()[var]
